### ISSUE TITLE
Image styles: "Desaturate" is "Grayscale" really.

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#1771]
![backdrop-image_style_preview-desaturate](https://cloud.githubusercontent.com/assets/2423362/14227423/9eadc0cc-f940-11e5-88b1-edcd9b36cb06.png)

It's not just the preview - the actual effect applied is grayscale.

**Comment by klonos** [comment]
...now, I have not used it recently, but it's the same with D7 and D8. I'm really surprised the nobody has complained about it!?

**Comment by klonos** [comment]
Was there ever an actual desaturate effect in previous versions of Drupal? We should:

1. rename the effect to "Grayscale" cause that's what it actually does
2. see if we can add an actual desaturate effect if possible

**Comment by Graham-72** [comment]
> I have not used it recently

I have never used it, nor ever thought of doing so. Likewise mirror/flip issue #1762
Why keep them? Just asking.

**Comment by klonos** [comment]
People find them useful + they serve as examples for contrib I guess.

**Comment by gifad** [comment]
@klonos : "Grayscale" applies to a color representation using a single channel;
"Desaturation" applies to an image transformation of a multiple channels image that equalize the three channels (in case of RVB model), or set the Saturation channel to 0 (in case of HSV model), etc...

So Desaturation is more exact : if you look at a desaturated image/jpeg in a photoshop like software, it's still a color image !

What shocks you is that it is a "Total Desaturation" effect, while you were waiting for a slider to apply a partial desaturation…

**Comment by klonos** [comment]
Since we do not provide any option to set the level of desaturation and we apply a "total desaturation" (=grayscale), I think we should be calling the thing what it is so that users know what to expect.

Besides:

https://github.com/backdrop/backdrop/blob/1.x/core/includes/image.inc#L364

```
/**
 * Converts an image to grayscale.
 *
```

https://github.com/backdrop/backdrop/blob/1.x/core/modules/system/image.gd.inc#L197

```
/**
 * Convert an image resource to grayscale.
 *
```

https://github.com/backdrop/backdrop/blob/1.x/core/modules/system/image.gd.inc#L215

```
return imagefilter($image->resource, IMG_FILTER_GRAYSCALE);
```

**Comment by gifad** [comment]
May I suggest that backdrop (and drupal, and php) may be wrong ?

```
$ exiftool ~/Sites/backdrop/sites/base/files/styles/desaturate/public/inline-images/2016/04/02/IMG_0335.preview.jpg 
ExifTool Version Number         : 9.98
File Name                       : IMG_0335.preview.jpg
……
File Type                       : JPEG
File Type Extension             : jpg
MIME Type                       : image/jpeg
JFIF Version                    : 1.01
Resolution Unit                 : None
X Resolution                    : 1
Y Resolution                    : 1
Comment                         : CREATOR: gd-jpeg v1.0 (using IJG JPEG v70), quality = 75.
Image Width                     : 240
Image Height                    : 160
Encoding Process                : Baseline DCT, Huffman coding
Bits Per Sample                 : 8
Color Components                : 3                 <<<<<<<<<<<<<<<<<<<<<<
Y Cb Cr Sub Sampling            : YCbCr4:2:0 (2 2)
Image Size                      : 240x160
Megapixels                      : 0.038
```
made with backdrop !
![capture d ecran 2016-04-02 a 19 46 53](https://cloud.githubusercontent.com/assets/15943462/14228112/571e843c-f90c-11e5-9cad-068a4771521b.jpg)

**Comment by klonos** [comment]
My point was that, technicalities and term definitions aside, we intent to display this word to the end users. These users might not be design or multimedia experts and all they see when they apply the effect is their image turn grayscale (some people might even call it "black and white" because they do not understand the difference). Also, they understand "grayscale" while they have to look up the word "desaturate". Even if they don't understand either of these terms, say they google both...

[grayscale](https://www.google.com/search?q=define+grayscale):
> a range of grey shades from white to black, as used in a monochrome display or printout.

...gray, shades of black and white. Sure.

[desaturate](https://www.google.com/search?q=define+desaturate)
> make unsaturated or less saturated

...explaining a a word with more derivatives of the same word. Great! ...lets try again:

[saturate](https://www.google.com/search?q=define+saturate)
> cause (something) to become thoroughly soaked with water or other liquid so that no more can be absorbed.

WTF? ...don't understand! Lets give it a go to see what it does... aaah! IT'S GRAYSCALE/B&W!!

Catch my drift now?

**Comment by bdanin** [comment]
I agree that using the word "desaturate" is a lot more confusing that "grayscale", because I was looking for something that could partially desaturate an image and this effect only does one function: 100% desaturation to grayscale

I am in favor of calling it "grayscale" for semantic clarity.
